api/src/main/java/com/thesurvey/api/controller/AuthenticationController.java

```diff
@@ -55,12 +50,9 @@ public ResponseEntity<UserResponseDto> register(
         @ApiResponse(code = 404, message = "요청한 리소스 찾을 수 없음")
     })
     @PostMapping("/login")
-    public ResponseEntity<UserResponseDto> login(@RequestBody UserLoginRequestDto userLoginRequestDto) {
-        try {
-            UserResponseDto userResponseDto = authenticationService.login(userLoginRequestDto);
-            return ResponseEntity.ok(userResponseDto);
-        } catch (AuthenticationException e) {
-            throw new ExceptionMapper(ErrorMessage.UNAUTHORIZED_REQUEST);
-        }
+    public ResponseEntity<?> login(
```

Comment:
As we're using `request` and `response` types, we should specify the object type.